docs/en/changes/changes.md

```diff
@@ -45,6 +45,7 @@
   instance.
 * Fix Zipkin trace query the max size of spans.
 * Add `tls` and `https` component IDs for Network Profiling.
+* Support Elasticsearch column alias for the compatibility between storage logicSharding model and no-logicSharding model.
```

Comment:
I think you'd better reword the change log item `* [Breaking Change] Many columns of metrics and records model names are changed, The H2/Mysql/Tidb/Postgres storage
  users are required to remove all metrics-related and records-related tables for OAP to re-create or use a new database
   instance.`

Reply:
Besides this, I would recommend writing a `FAQ - new ElasticSearch storage option explanation in 9.2.0` to guide users and link from change log and storage docs.